wireguard.rst

```diff
@@ -19,230 +19,80 @@ Features:
-  reload_config
-  ifdown wg1; ifup wg1
-
-You can use the same API to change the configuration of the server instance.
-
-Add a new account (peer)
-------------------------
-
-Create a new account, ensuring the ``account`` field is unique within the same instance:
-
-.. code-block:: bash
-
-  echo '{"enabled": true, "instance": "wg1", "account": "user1", "route_all_traffic": false, "client_to_client": false, "ns_routes": [], "preshared_key": true}' | /usr/libexec/rpcd/ns.wireguard call set-peer
-  
-Options:
-
-- ``route_all_traffic``: if set to ``true``, when the client connects, it will send all the traffic to the server.
-  This is useful for ensuring all client traffic is encrypted and routed through the VPN. This flag should be set only if the client
-  must access the internet through the VPN.
-- ``client_to_client``: if set to ``true``, the client will be able to communicate with all other peers and not only with the server.
-  This is useful for enabling communication between clients in a mesh network configuration.
-- ``preshared_key``: if set to ``true``, automatically create a pre-shared key that will be used in the peer downloaded configuration.
-  This adds an additional layer of security by requiring a shared secret for communication.
-- ``ns_routes``: a list of network CIDR, automatically routes the networks to this peer;
-  this is used for site-to-site (net2net) connections, allowing the server to access multiple remote networks through the VPN.
-
-Save and apply:
-
-.. code-block:: bash
-
-  uci commit network
-  reload_config
-  ifdown wg1; ifup wg1
-
-
-Download the account configuration
-----------------------------------
-
-The account configuration can be downloaded both in text format or a QR code and is suitable to be imported in a WireGuard client.
-
-Download the text format if you want to configure a Linux machine or another NethSecurity:
-
-.. code-block:: bash
-
-  echo '{"instance": "wg1", "account": "user1"}' |  /usr/libexec/rpcd/ns.wireguard call download-peer-config | jq -r .config
+- `Status`: enable or disable the peer
+- `Name`: the name of the peer
+- `Reserved IP`: the static IP address that will be assigned to the peer, must be inside the VPN network, it will be pre-filled with the next available IP
+- `Pre-shared key`: if enabled, a pre-shared key will be automatically created to enhance security
+- `Route all traffic`: if enabled, when the client connects, it will send all the traffic to the server
+- `Server networks`: which networks the peer can access, all LAN networks will be automatically added
+- `Peer networks`: networks reachable on the peer side
 
-Output example: ::
+.. note::
 
-  # Account: user1 for wg1
-  [Interface]
-  PrivateKey = iGn1xg3pENbVCJpJWf4EqOYtNu7HZj1dg5iIX9AU0FY=
-  Address = 10.98.95.2
-  # Custom DNS disabled
+  It's possible to create a client-to-site connection by leaving empty the `Peer networks` entries. This will allow the client to access the server networks.
```

Comment:
```suggestion
  It's possible to create a client-to-site (Road Warrior) connection by leaving empty the `Peer networks` entries. This will allow the client to access the server networks.
```